### PR TITLE
[ALLUXIO-1871 ] Load env variables again after overriding ALLUXIO_RAM_FOLDER

### PR DESCRIPTION
**Pull request by peisun1115**
Small style fix in alluxio-start.sh

### CODE REVIEW

**Comment by peisun1115** [comment]
The bugfix has been included in the other PR. https://github.com/Alluxio/alluxio/pull/3014

The only diff here is just one more bash style change. Let me know if you prefer to just reverting this.

File: bin/alluxio-start.sh [hunk 1]
```diff
@@ -52,7 +52,8 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      if ! mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null; then
+      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null; then
```
Comment (uronce-cc) [inline bin/alluxio-start.sh]: redundant `; then` at the end

Comment (peisun1115) [inline bin/alluxio-start.sh]: done

**Comment by apc999** [comment]
thanks for the fix. LGTM

**Pull request merged**